packages/html2sb-compiler/toScrapbox.js

```diff
@@ -11,6 +11,9 @@ function toSimpleText (node) {
     var src = '';
     if (node.src) {
       src = node.src;
+      if (!/^https?:\/\/gyazo.com\//.test(src) && !/\.(png|jpe?g|gif|svg)$/.test(src)) {
```

Comment:
```suggestion
      if (new URL(src).hostname !== 'gyazo.com' && !/\.(png|jpe?g|gif|svg|webp)$/.test(src)) {
```

Reply:
add webp extension and use URL class

Reply:
some times the src is relative URL
eg: `test.resources/47C4AAC0-EFA4-4E4F-98F5-0ACD38B650F8.png`

and this makes exception at `new URL()`